bigchaindb/upsert_validator/validator_election.py

```diff
@@ -32,15 +37,15 @@ def __init__(self, operation, asset, inputs, outputs,
         super().__init__(operation, asset, inputs, outputs, metadata, version, hash_id)
 
     @classmethod
-    def current_validators(cls, bigchain):
+    def current_validators(cls, bigchain, height=None):
```

Comment:
This method should belong to the `BigchainDB` class, what do you think?

Reply:
The `BigchainDB` already has a method `get_validators` this function is to just type cast the incomptabile formats of base58 and base64 public keys.

Reply:
Can we make `BigchainDB::get_validators` return `base58` instead of `base64`? AFAIK `get_validators` is only used for changing the validator set.

I think we are still in time to change the interface. What do you think?

Reply:
`get_validators` is used to list current validators in the `/api/v1/validators` API. We use base64 here so that users can actually cross verify the values in their Tendermint configs which are also base64 encoded.

Reply:
Then why not adding an `encoding` parameter to `get_validators`? Also, `current_validators` should be called differently since it accepts a `height` parameter. `get_validators` would be a good name, that is the same we have in the `BigchainDB` class, that's why I'm pushing to use that methods. Moreover, this method is just taking the result of the method of another class, encoding the result in another way.

Reply:
> Then why not adding an encoding parameter to `get_validators` ? 

I imagine that would be one of the tasks when standardizing the interface.

> Also, current_validators should be called differently since it accepts a height parameter. get_validators would be a good name, that is the same we have in the BigchainDB class, that's why I'm pushing to use that methods. Moreover, this method is just taking the result of the method of another class, encoding the result in another way.

:+1: